src/lib/helmet.js

```diff
@@ -1,20 +1,13 @@
 //Extracted from common-express and modified
 
-const {
-  generateNonce,
-} = require("@govuk-one-login/di-ipv-cri-common-express/src/lib/strings.js");
-
 module.exports = {
   contentSecurityPolicy: {
     directives: {
       defaultSrc: ["'self'"],
       styleSrc: ["'self'"],
       scriptSrc: [
         "'self'",
-        (req, res) => {
-          res.locals.cspNonce = res.locals.cspNonce || generateNonce();
```

Comment:
Have we tested the nonce generation? Just want to confirm this line on 15 was actually defaulting to the generateNonce method